legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/relationalExtension.pure

```diff
@@ -113,11 +113,21 @@ function meta::relational::functions::typeInference::inferRelationalType(rop: Re
 }
 
 function meta::relational::functions::typeInference::inferRelationalType(rop: RelationalOperationElement[1], failOnMatchFailure:Boolean[1]):meta::relational::metamodel::datatype::DataType[0..1]
+{
+  meta::relational::functions::typeInference::inferRelationalType($rop, $failOnMatchFailure, false);
+}
+
+function meta::relational::functions::typeInference::inferRelationalTypeExceptForDynaFunction(rop: RelationalOperationElement[1], failOnMatchFailure:Boolean[1]):meta::relational::metamodel::datatype::DataType[0..1]
```

Comment:
make it private

Reply:
inferRelationalTypeExceptForDynaFunction should be private?

Reply:
oh ok, i can't be as it is used in separate file